config-builder/src/main/java/com/ca/apim/gateway/cagatewayconfig/bundle/builder/BundleMetadataBuilder.java

```diff
@@ -93,7 +93,7 @@ private BundleMetadata buildFullBundleMetadata (final List<Entity> entities,
 
     private Collection<Metadata> getDefinedEntitiesMetadata(final List<Entity> definedEntities) {
         return definedEntities.stream().filter(FILTER_NON_ENV_ENTITIES_EXCLUDING_FOLDER)
-                .map(Entity::getMetadata).collect(Collectors.toList());
```

Comment:
remove getMetadata method from Entity if its not used

Reply:
@uppoju Entity getMetadata method is needed for Env entities